bundles/python/python_lexer.moon

```diff
@@ -100,4 +140,14 @@ howl.aux.lpeg_lexer ->
       identifier,
       decorator
     }
+
+    f_string: any f_strings
+
+    string: any {
+      f_prefix * V'f_string'
+      string
+    }
   }
+
+  rules[k] = v for k, v in pairs interpolations
```

Comment:
@shalabhc Because the rules recurse into each other with `V`. Just like in the MoonScript one.